### ISSUE TITLE
Python `get_color_rgba` returns an empty image

### ISSUE THREAD

**Issue by yamokosk** [haosulab/SAPIEN-Release#15]
Using the `test_point_cloud.py` as an example to build my own test, the image saved for my scene is just a empty image. Is the camera API broken?

**Comment by fbxiang** [comment]
The camera API should work. Here are somethings to check:
1. Did you call take_picture before get_color_rgba? 
2. Make sure the camera is not pointing at empty space.
If you still cannot figure out, you can show us some simple code that reproduce your problem.

**Comment by yamokosk** [comment]
1. Yes, I reproduced all of the API calls that I found in `test_point_cloud.py`, including `take_picture`. 
2. For the camera pose I copied the values i was using to set the pose of the render controller's camera.
3. Yes, my scene has a light source.

I suspect that something is not right with my camera pose. In some testing, I added some print statements to some of the steps in my code that converts the desired pose from (x,y,z,roll,pitch,yaw) and saw something concerning:

```
mat4x4 = self._config.pose.as_transform_matrix()
print('mat4x4: {}'.format(mat4x4))
spose = sapien.Pose.from_transformation_matrix(mat4x4)
self._camera_mount_actor.set_pose(spose)
````

Notice the values output by the Sapien pose object:

```
mat4x4: [[-1.0000000e+00  0.0000000e+00  1.2246468e-16  4.5000000e+00]
 [ 0.0000000e+00  1.0000000e+00  0.0000000e+00  0.0000000e+00]
 [-1.2246468e-16  0.0000000e+00 -1.0000000e+00  2.3000000e+00]
 [ 0.0000000e+00  0.0000000e+00  0.0000000e+00  1.0000000e+00]]
spose: Pose([4.5, 0, 2.3], [0, -nan, inf, -nan])
```

`mat4x4` is a valid transformation matrix and computed from:

```
mat4x4 = np.eye(4)
mat4x4[:3,3] = np.array([4.5, 0.0, 2.3])
mat4x4[:3,:3] = Rotation.from_euler('xyz', 
            [0.0, 180.0, 0.0],
            degrees=True).as_matrix()
```

Sapien's rotation matrix to quaternion code is not properly checking for an edge case (https://github.com/haosulab/SAPIEN-Release/blob/master/src/utils/pose.hpp#L14). See Eigen's implementation for reference: https://gitlab.com/libeigen/eigen/-/blob/master/Eigen/src/Geometry/Quaternion.h#L826

But aside from this, looks like I can specify the quaternion directly instead  of relying on Sapien's conversion. What is the image axis of the camera? -Z axis? Z-axis?

**Comment by yamokosk** [comment]
Fixed! After specifying the quaternion directly, I'm getting images now. Thanks for the tips!

**Comment by fbxiang** [comment]
Thanks for the report, we will have it fixed soon.

**Comment by fbxiang** [comment]
Should be fixed in 1.0
